### ISSUE TITLE
packbeam --external requires full path

### ISSUE THREAD

**Issue by pguyot** [atomvm/atomvm_rebar3_plugin#29]
Surprisingly, it seems that

```shell
rebar3 atomvm packbeam -e estdlib/estdlib.avm -e eavmlib/eavmlib.avm
```

doesn't work and fails with enoent, while:

```shell
rebar3 atomvm packbeam -e `pwd`/estdlib/estdlib.avm -e `pwd`/eavmlib/eavmlib.avm
```

does work.

I guess the current working directory is not where rebar3 is invoked from. It should at least be mentioned in documentation.

**Comment by UncleGrumpy** [comment]
Indeed. It is only vaguely implied in the example of packing an external file… But not stated explicitly. I will change the help to say this should be the full path to the avm file, and make it clear in the documentation as well.

**Comment by UncleGrumpy** [comment]
Please take a look at PR #36, it enables using relative paths as well for external AVMs.
